CONTRIBUTING.md

```diff
@@ -128,42 +120,47 @@ submit a pull request:
    to other methods available in pydicom.
 
 -  Documentation and high-coverage tests are necessary for enhancements to be
-   accepted. Bug-fixes shall be provided with 
+   accepted. Bug-fixes shall be provided with
    [regression tests](https://en.wikipedia.org/wiki/regression_testing) that
    fail before the fix. For new features, the correct behavior shall be
-   verified by feature tests. A good practice to write sufficient tests is 
+   verified by feature tests. A good practice to write sufficient tests is
    [test-driven development](https://en.wikipedia.org/wiki/Test-driven_development).
 
 You can also check for common programming errors and style issues with the
 following tools:
 
--  Code with good unittest **coverage** (current coverage or better), check
- with:
+-  The test coverage should be at least as good as the current coverage and any
+new lines of code must be covered:
 
   ```bash
   $ pip install pytest pytest-cov
-  $ py.test --cov=pydicom path/to/test_for_package
+  $ cd pydicom/
+  $ py.test --cov=pydicom tests/
   ```
 
--  No pyflakes warnings, check with:
+-  No type hint errors:
 
   ```bash
-  $ pip install pyflakes
-  $ pyflakes path/to/module.py
+  $ pip install mypy
+  $ cd pydicom/
+  $ mypy
   ```
 
--  No PEP8 warnings, check with:
+-  No style or code warnings
 
   ```bash
-  $ pip install pycodestyle  # formerly called pep8 
-  $ pycodestyle path/to/module.py
+  $ pip install ruff
+  $ cd pydicom/
+  $ ruff check .
   ```
 
--  AutoPEP8 can help you fix some of the easy redundant errors:
+Finally, your contribution should be [blackened](https://github.com/psf/black)::
 
   ```bash
-  $ pip install autopep8
-  $ autopep8 path/to/pep8.py
+  $ pip install black
+  $ cd pydicom/
+  $ black src/pydicom/
+  $ black tests/
   ```
```

Comment:
Maybe add using `precommit` as an alternative to these. This would also take care of any changes we make in the pre-commit config.

Reply:
Done, also bumped the package version in the pre-commit config to match `pyproject.toml`